modules/python/test/test_misc.py

```diff
@@ -282,12 +282,11 @@ def test_parse_to_int64_convertible(self):
 
     def test_parse_to_int64_not_convertible(self):
         min_int64, max_int64 = get_limits(ctypes.c_longlong)
-        for not_convertible in (1.2, np.float(4), float(3), np.double(45), 's', 'str',
+        for not_convertible in (1.2, float(3), np.double(45), 's', 'str',
                                 np.array([1, 2]), (1,), [1, 2], min_int64 - 1, max_int64 + 1,
                                 complex(1, 1), complex(imag=2), complex(1.1), np.bool_(True),
                                 True, False, np.float32(2.3), np.array([3, ], dtype=int),
-                                np.array([-2, ], dtype=np.int32), np.array([1, ], dtype=np.int),
-                                np.array([11, ], dtype=np.uint8)):
+                                np.array([-2, ], dtype=np.int32), np.array([11, ], dtype=np.uint8)):
```

Comment:
`np.int32`, `np.iint8`

Reply:
np.array([1, ], dtype=np.int) is np.array([1, ], dtype=int) which is already tested as np.array([3, ], dtype=int), so I removed it.